docs/vhlo.md

```diff
@@ -152,7 +152,7 @@ for forward and backward compatibility testing.
 ```bash
 $ export TARGET_VERSION=0_X_0
 $ cp stablehlo/tests/stablehlo_legalize_to_vhlo.mlir stablehlo/tests/stablehlo_legalize_to_vhlo.$TARGET_VERSION.mlir
-$ build/bin/stablehlo-translate --serialize stablehlo/tests/stablehlo_legalize_to_vhlo.$TARGET_VERSION.mlir --target=current --strip-debuginfo > stablehlo/tests/stablehlo_legalize_to_vhlo.$TARGET_VERSION.mlir.bc
+$ build/bin/stablehlo-translate --serialize --target=$TARGET_VERSION --strip-debuginfo stablehlo/tests/stablehlo_legalize_to_vhlo.$TARGET_VERSION.mlir > stablehlo/tests/stablehlo_legalize_to_vhlo.$TARGET_VERSION.mlir.bc
```

Comment:
This would probably fail too: `--target=0_10_0` would need to be `0.10.0`.

how about:

```bash
$ export TARGET_VERSION=0.X.0
$ export TARGET_FILENAME=${TARGET_VERSION//./_}
$ cp stablehlo/tests/stablehlo_legalize_to_vhlo.mlir stablehlo/tests/stablehlo_legalize_to_vhlo.$TARGET_FILENAME.mlir
$ build/bin/stablehlo-translate --serialize --target=$TARGET_VERSION --strip-debuginfo stablehlo/tests/stablehlo_legalize_to_vhlo.$TARGET_FILENAME.mlir > stablehlo/tests/stablehlo_legalize_to_vhlo.$TARGET_FILENAME.mlir.bc
...

```

Reply:
Good catch! Done.